### ISSUE TITLE
Revive after Reconnect

### ISSUE THREAD

**Issue by ErrorRExorY** [esx-framework/esx_ambulancejob#54]
Hey Guys, we're using the latest version of ESX_AmbulanceJob and we have the problem, that when a User Disconnects and reconnects, the Script doesn't recognize the Player as dead and we can't revive them via Ambulance Revive job

**Comment by Gellipapa** [comment]
@ErrorRExorY Hi! Do you use latest version 1.10.4? https://github.com/esx-framework/esx_core/releases/download/1.10.4/esx_core.zip We fixed this problem this hotfix release. You should download and i hope working.

**Comment by ErrorRExorY** [comment]
Hey @Gellipapa 
Thank you for the fast response and no we oversaw that. Unfortunately we modified the code alot and changing the core, because most changes were made in es_extended from us, its too much work for us! But I tried it on a clean server and the error was fixed. 

Is there any way for me to look at the changes for that and try to make the changes that were made in the fix instead of reimplementing all of our code back into a new core?

Kind regards 
ErroRExorY
